src/templates/standard/package.json

```diff
@@ -5,44 +5,22 @@
   "author": "",
   "scripts": {
     "start": "npm run build && npm run serve",
-    "dev": "better-npm-run development",
-    "build": "better-npm-run create-build",
-    "serve": "better-npm-run run-server",
+    "dev": "cross-env NODE_ENV=development MY_CUSTOM_ENV=foo dev-toolkit watch",
```

Comment:
👍 for migrating this to a more _node_ way of doing this!

Reply:
yeah, I thought giving people better-npm-run by default is more overhead than necessary